Yoast/javascript
# Dutch morphology small fixes and clean-up

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Introduce small fixes to the Dutch stemming and clean up the code

## Test instructions

This PR can be tested by following these steps:

* Run yarn test on the branch and make sure all tests pass

## Quality assurance
